src/en/authors-charm-policy.md

```diff
@@ -4,11 +4,11 @@ Title: Charm store policy
 
 This document serves to record the policies around charms and bundles included
 in the charm store, and the management of said collection. Charms and bundles in
-the store are peer reviewed by the community and are considered ready for
+the store are peer-reviewed by the community and are considered ready for
 production grade consumption. These charms are marked as reviewed in the GUI and must follow
 these quality guidelines in ordered to be considered for the Store, otherwise they can live in personal name spaces.
```

Comment:
All lines should not be longer than around 80 characters. My (vim) editor has a maximum length of 79 apparently.

Exceptions: do not break up links (including reference furniture). Doing so supposedly puts stress on the Markdown:HTML conversion. The following should be put on one line (I haven't seen any such links here; this is an FYI):
[blah blah blah](looooooooooooooooooooooooooooooongggggggggggggggggggggggggggggggggggggggggish link)